libraries/botbuilder-expression/src/buildInFunction.ts

```diff
@@ -1293,6 +1626,62 @@ export class BuiltInFunctions {
                 ReturnType.Number,
                 BuiltInFunctions.ValidateBinaryNumber),
             new ExpressionEvaluator(ExpressionType.CreateArray, BuiltInFunctions.Apply((args: ReadonlyArray<any>) => Array.from(args)), ReturnType.Object),
+            new ExpressionEvaluator(
+                ExpressionType.Array,
+                BuiltInFunctions.Apply((args: ReadonlyArray<any>) => [args[0]], BuiltInFunctions.VerifyString),
+                ReturnType.Object,
+                BuiltInFunctions.ValidateUnary),
+            new ExpressionEvaluator(
+                ExpressionType.Binary,
+                BuiltInFunctions.Apply((args: ReadonlyArray<any>) => this.ToBinary(args[0]), BuiltInFunctions.VerifyString),
+                ReturnType.String,
+                BuiltInFunctions.ValidateUnary),
+            new ExpressionEvaluator(
+                ExpressionType.DataUri,
+                BuiltInFunctions.Apply(
+                    (args: Readonly<any>) => 'data:text/plain;charset=utf-8;base64,'.concat(Buffer.from(args[0]).toString('base64')), BuiltInFunctions.VerifyString),
```

Comment:
personal opinion， is there a exist library to handle `datauri`, `dataUriToBinary` and `dataUriToString`. Maybe we can use a library to do more param checks. can [datauri](https://www.npmjs.com/package/datauri) help?

Reply:
datauri package accept a file path as input which is not consistent with the one in WDL doc https://docs.microsoft.com/en-us/azure/logic-apps/workflow-definition-language-functions-reference#dataUri.